Updated icons

smombartz opened the pull request:
redrawn, sprakly clean code and neatly indented. phew.

julien51 commented:
Please generate the components and put the screenshots here ;)

smombartz commented:
created svg components and uploaded screenshot

julien51 commented:
Please revise against master and push again. You should now be able to use storybook to make a screenshot of all SVG at once!

Thanks

smombartz commented:
updated icons, screenshot, and snapshot.
@julien51 i added a new menu-hamburger and menu-chevron icons – you want to update the links to those files?

there is an issue with the unlock icon, and not sure why its showing up funky in storybook. file looks alright...
<img width="1416" alt="screen shot 2018-11-30 at 14 08 43" src="https://user-images.githubusercontent.com/1377389/49309612-cd8cb080-f4a9-11e8-8247-b78fd0140e4a.png">

julien51 commented:
I know what's going on, I'll fix in another PR!

LG (you can merge!)

smombartz commented:
updated unlock brand images (and updated screenshot) – all squiggly clean, consistent and and in order!

new wordmark svg is now black too – new brand color is #fs6771

over to you @julien51

julien51 commented:
> new wordmark svg is now black too – new brand color is #fs6771

That does not seem to be a color :/
